src/net.cpp

```diff
@@ -1675,13 +1678,15 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
         // Only connect out to one peer per network group (/16 for IPv4).
         int nOutboundFullRelay = 0;
         int nOutboundBlockRelay = 0;
+        int nOutboundFullRBF = 0;
```

Comment:
nit: why using ugly variable name in new code?

Reply:
What do you suggest?  The variable matches with the naming in the previous two lines.